local-volume/README.md

```diff
@@ -17,7 +17,27 @@ directories by creating and cleaning up PersistentVolumes for each volume.
 
 ## Feature Status
 
-Current status: 1.7 & 1.8 - Alpha
+### 1.9: Alpha
+
+**Important:** Both `PersistentLocalVolumes` and `VolumeScheduling` feature gates
+must be enabled starting in 1.9.
+
+What works:
+* Everything in 1.7.
+* Multiple local PVCs in a single pod.
```

Comment:
We might want to put this bullet and the next bullet together, since I believe the features are linked together. Plus we could link to the fact that a storage class with a special parameter is also required, as described later in the document.